metabase/metabase
# Allowing users to hide logo on embedding

Issue: #27293

Allow users to disable/hide logo in embedding by the param `logo` (defaults to `true`)

#### How to test
1. Run Metabase with this branch as an enterprise user (port 3000)
1. Configure enterprise token
1. Enable JWT authentication and make sure you set the secret to `ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` https://github.com/metabase/sso-examples/blob/master/app-embed-example/server.js#L9-L10
1. Clone https://github.com/metabase/sso-examples
1. Go to `/app-embed-example/`
1. Run `yarn dev` (it will start port 3001 and 3002)
1. Go to http://localhost:3001/analytics and test if embedding shows breadcrumbs as desired.
1. You could try append different parameters to see it in action e.g.
    1. `?logo=false`
    2. `?logo=false&side_nav=true`
    3. `?logo=false&breadcrumbs=false`
    4. `?breadcrumbs=false` 
1. Test mobile view.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27391)
